web/impact/impact/v1/views/office_hours_calendar_view.py

```diff
@@ -118,50 +208,83 @@ def _get_office_hours_data(self):
         self.response_elements['location_choices'] = self.location_choices()
         program_families = self.mentor_program_families()
         self.response_elements['mentor_program_families'] = program_families
+        self.response_elements['user_startups'] = self._user_startups()
         self.succeed()
 
+    def _user_startups(self):
+        return self.target_user.startupteammember_set.order_by(
+            '-id').values(
+                "id",
```

Comment:
from QA:
`id` should be the `startup__id` and this list should also be `distinct()`

Reply:
good catch! Fixed.